graphql-test-utils/create-graphql-mock-server.js

```diff
@@ -0,0 +1,69 @@
+import { graphql, buildClientSchema } from 'graphql';
+import { addMockFunctionsToSchema } from 'graphql-tools';
+import { formatApolloErrors } from 'apollo-server-errors';
+import loadSchema from './load-schema';
+
+const getRootValue = (operations, operationName, variables) => {
+  if (!operationName || !operations[operationName]) {
+    return {};
+  }
+  const op = operations[operationName];
+  if (typeof op === 'function') {
+    try {
+      return op(variables);
+    } catch (e) {
+      return e; // properly handle dynamic throw new GraphQLError("message")
+    }
+  }
+  return op;
+};
```

Comment:
Nit and subjective :) this would have helped me:

```js
const executeGraphQlOperation = (operation, variables) => {
   if (typeof operation === "function") {
      try {
         return operation(variables);
      } catch (e) {
         // TODO: Properly handle dynamic throw new GraphQLError("message")
         return e; 
      }
   }

   return operation
};
const getRootValue = (operations, operationName, variables) => {
   if (!operationName || !operations[operationName]) {
      return {};
   }

   const operation = operations[operationName];

   return executeGraphQlOperation(operation, variables);
};

```

Reply:
Yeah I took this from here: https://github.com/tgriesser/cypress-graphql-mock/blob/6000875ebc9a16f0a3a20fbef72a2c41f0379eca/src/index.ts#L194-L212

I can apply your suggestion though, thanks

Reply:
Ah kay. Doesn't matter too much. Just what my mind translated the code to :)